test/cases/regression-excluded-dependency.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+# This test case verifies that a blueprint can include a package which has a
+# dependency that is listed among "excluded" for a certain image type and
+# osbuild-composer doesn't fail to depsolve this blueprint.
+#
+# The script currently works only for RHEL and CentOS which provide
+# "redhat-lsb-core" package and exclude "nss" package in the image type
+# definition. The testing contains the "redhat-lsb-core" package which can only
+# be installed if the "nss" package isn't excluded
+#
+# Bug report: https://github.com/osbuild/osbuild-composer/issues/921
+
+# NOTE: ONLY WORKS IN RHEL 8.5
+# Get OS data.
+source /etc/os-release
+
+# Provision the software under tet.
+/usr/libexec/osbuild-composer-test/provision.sh
+
+if [[ "${ID}-${VERSION_ID}" != "rhel-8.5" ]]; then
```

Comment:
It would be great to have this future-proof and run this on rhel >= 8.5. Please, do not solve it in this PR, this is just a reminder for us that we should think about this.